runner/status.go

```diff
@@ -26,12 +26,12 @@ const (
 	ABORTED
 	// Operation timed out and was killed.
 	TIMEDOUT
-	// Invalid or error'd request. Original runner state not affected. Retry may work after mutation.
+	// Request rejected for reasons unrelated to the user.
```

Comment:
I had always thought BADREQUEST was 'bad task description: invalid os command, invalid snapshot id, etc' and the FAILED was 'an error other than BADREQUEST: typically due to missing support service, bug in the platform code, etc.  example: error creating directories used by the runner, error communicating with bundlestore, etc.'  (Note: so my understanding/description of FAILED vs BADREQUEST appears to be opposite of what was implemented... I can adjust my understanding.)